sympy/physics/control/lti.py

```diff
@@ -0,0 +1,655 @@
+from sympy import Basic, Mul, degree, Symbol, expand, cancel, Expr, exp
+from sympy.core.evalf import EvalfMixin
+from sympy.core.numbers import Integer
+from sympy.core.sympify import sympify, _sympify
+
+__all__ = ['TransferFunction', 'Series', 'Parallel', 'Feedback']
+
+
+class TransferFunction(Basic, EvalfMixin):
+    """TransferFunction(num, den, var)
+
+    A class for representing continuous transfer functions. This class is used to represent
+    LTI (Linear, time-invariant) systems in transfer function form. The arguments
```

Comment:
I think this needs more specifity. It represents LTI systems that can be strictly described by ratios of polynomials in the Laplace Transform complex variable.